spec/spec_helper.rb

```diff
@@ -1,7 +1,26 @@
 # frozen_string_literal: true
 
-require 'bundler/setup'
-require 'activeadmin-async_exporter'
+ENV['RAILS_ENV'] ||= 'test'
+require 'dummy/config/environment.rb'
+require 'pry'
+require 'active_admin'
+require 'timecop'
+
+require 'simplecov'
+SimpleCov.start
+
+Rails.application.routes.default_url_options[:host] = 'localhost:3000'
+
+require 'tmpdir'
+ActiveStorage::Blob.service =
+  ActiveStorage::Service::DiskService.new(root: Dir.mktmpdir('active_storage_tests'))
```

Comment:
what's this for?

Reply:
This should be handled via the `storage.yml` no?